api-first-approach/index.asciidoc

```diff
@@ -0,0 +1,90 @@
+= API first approach with OpenAPI generator
```

Comment:
The title/subtitle is missing the info that it is a quarkus tutorial.

Reply:
It is actually a 'general' tutorial. I did use Quarkus, but you can do it the same way in Spring.
I added some information about it in the description, but I would leave the title as it is.